website/content/partials/known-issues/expiration-metrics-fatal-error.mdx

```diff
@@ -0,0 +1,20 @@
+### Fatal error during expiration metrics gathering causing Vault crash
+
+#### Affected versions
+
+This issue affects Vault Community and Enterprise versions:
+- 1.13.9
+- 1.14.5
+- 1.15.1
+
+A fix will be issued in Vault 1.13.10, 1.14.6, and 1.15.2.
+
+#### Issue
+
+A recent change to Vault to improve state change speed (e.g. becoming active or standby) introduced a concurrency issue
+which can lead to a concurrent iteration and write on a map, causing a fatal error and crashing Vault. This error occurs
+when gathering lease and token metrics from the expiration manager.
+
+There is no workaround.
```

Comment:
My understanding is that the active node would crash, and in an HA cluster another node would take over. It would be vulnerable to the same bug, but it might or might not encounter it immediately.

While there isn't a workaround, there is a mitigation of sorts. If my understanding is kind of correct, I think it might be nice to soften this language and maybe make it a little more clear that this shouldn't happen constantly?

Reply:
I believe your understanding to be correct regarding another node taking over active duties. I'll clarify more around the failure state here in that the Vault node would crash, but not Vault as a system (in a HA environment).